### ISSUE TITLE
Unable to get bounding boxes for test images

### ISSUE THREAD

**Issue by Rathna21** [allanzelener/YAD2K#162]
Thanks for the script. I successfully converted the YOLO 2 VOC model to Keras model. 

While testing, I am unable to get bounding boxes for the test images in images folder and I get output as "Found 0 boxes in dog.png" and same for other images.

Can someone help me on this?

**Comment by JesseLv001** [comment]
I have suffered the same problem while using my own dataset, the loss when training looks good but can not find boxes while test and draw the image. Can anybody solve it?

**Comment by Rathna21** [comment]
Adding to the first comment on "Found 0 boxes in dog.png", when i convert the model into keras and then when i try to load it and print the predicted result, I find that all the values are **NaN**

**Comment by DLeeeeeee** [comment]
I found that there are several BN layers whose variance contain negative number after converting. I am quite sure it is the reason to cause NaN output but I don't get how come these negatives exist. An interesting point is that negative number is always the first one in variance weights. So I guess we need to read the first variance number differently?
